.github/workflows/main.yml

```
@@ -57,7 +57,7 @@ jobs:
           python-version: "3.9"
 
       - name: "Install poetry, check-wheel-content, and twine"
-        run: "python -m pip install wheel twine check-wheel-contents"
+        run: "python -m pip install check-wheel-contents setuptools_scm twine wheel"
       - name: "Build package"
         run: "python setup.py sdist bdist_wheel"
```

```suggestion
        run: "python -m build"
```
The officially recommended approach that builds a tarball, then creates a wheel from the tarball.
It avoids to make a silly error when a file required by wheel is not included in the tarball.